src/runtime/test/recipe-resolver-test.js

```diff
@@ -24,24 +23,7 @@ describe('RecipeResolver', function() {
     return manifest.recipes[0];
   };
 
-  const createArc = () => {
-    return new Arc({
-      id: 'test',
-      slotComposer: {
-        modality: Modality.forName('dom'),
-        getAvailableContexts: (() => {
-          return [{
-            name: 'root',
-            id: 'r0',
-            tags: ['#root'],
-            handles: [],
-            handleConnections: [],
-            spec: {isSet: false}
-          }];
-        })
-      }
-    });
-  };
+  const createArc = () => new Arc({id: 'test', slotComposer: new FakeSlotComposer()});
```

Comment:
add `loader` and other required parameters.  Otherwise this will make it harder to convert to typescript.

Reply:
done